src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1217,6 +1218,9 @@ public void run() {
 					y = 1.05;
 					// ascend stairs
 				}
+				if (car.getFallDistance() > 1.5) { //Prevents Fall Distance stacking up causing fall-damage when climbing longer slopes
+					y = y*0.95;
```

Comment:
Does this still provide reasonable fall damage for a large fall?

Reply:
This part of the code is only executed whilst climbing blocks - it has no effect on normal fall damage, just fixes the issue with fall damage being dealt to the player whilst driving up long slopes